scripts/linters/pylint_extensions.py

```diff
@@ -81,6 +81,13 @@ class ExplicitKeywordArgsChecker(checkers.BaseChecker):
             'non-explicit-keyword-args',
             'All keyword arguments should be explicitly named in function call.'
         ),
+        'C0027': (
```

Comment:
Do you think this way it will be harder for anyone to find the next id for the new lint test they are going to add? If yes, do you have any idea how we can make it better? (I can't think of a way to make it simpler.)

Reply:
May be have a top level comment at the top of the file indicating the next ID.

Reply:
Added a top level comment.